static/views/admin/settings.php

```diff
@@ -726,11 +726,6 @@ class="wpp_settings_block"><?php printf(__('Get %1s image data.', ud_get_wp_prop
             <pre id="wpp_show_settings_array_result" class="wpp_class_pre hidden"></pre>
           </div>
 
-          <div class="wpp_settings_block">
```

Comment:
You are right to remove this but would be good to have a "clear cache" function somewhere to purge any cache related (e.g. object cache) stuff related to WPP.